zebra-network/src/protocol/external/arbitrary.rs

```diff
@@ -52,3 +52,16 @@ impl Arbitrary for InventoryHash {
 
     type Strategy = BoxedStrategy<Self>;
 }
+
+#[cfg(any(test, feature = "proptest-impl"))]
+impl Arbitrary for PeerServices {
+    type Parameters = ();
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
+        any::<u64>()
+            .prop_map(PeerServices::from_bits_truncate)
```

Comment:
Previously, `derive(Arbitrary)` was putting any `u64` value in these bits, which caused spurious errors.